tutormfe/patches/openedx-lms-production-settings

```diff
@@ -72,7 +72,7 @@ MFE_CONFIG["ACCOUNT_PROFILE_URL"] = "{% if ENABLE_HTTPS %}https://{% else %}http
 
 {% if get_mfe("communications") %}
 COMMUNICATIONS_MICROFRONTEND_URL = "{% if ENABLE_HTTPS %}https://{% else %}http://{% endif %}{{ MFE_HOST }}/communications"
-MFE_CONFIG["SCHEDULE_EMAIL_SECTION"] = True
+MFE_CONFIG["SCHEDULE_EMAIL_SECTION"] = "true"
```

Comment:
Why are we replacing bool to string here? We are not comparing anywhere it with "true" [here](https://github.com/search?q=repo%3Aopenedx%2Ffrontend-app-communications++SCHEDULE_EMAIL_SECTION&type=code).

Reply:
Thanks a lot for the review. I changed all the bools with strings in this PR because they would all trip with the same issue with `process.env` returning strings and MFE_CONFIG returning bool.
I might have overreached as I did not test all of them.

Given the latest round of comments in the course-authoring PR I think we might close this PR altogether. I'll update this PR after we settle the discussion there.

Reply:
I have gone through the course-authoring discussion and just got that the issue was in Quince 3. Is that resolved now so I can close this PR?

Reply:
The reason for replacing all bool to strings is that this would be the _theoretically_ correct behavior for all MFE_CONFIG variables.  I'll reproduce what I wrote on another PR:

> `getConfig()` still returns values as they were set by the .env files, unless they were overriden by runtime config. Since .env files are still supported, and the dotenv package that reads them in doesn't support booleans, we have no choice but to assume everything that `getConfig()` spits out is a string.

The problem is that some MFEs don't follow this assumption correctly.  For instance, to use your example search, take a look here:

https://github.com/openedx/frontend-app-communications/blob/9279a3e4ce888f6cc198ecb63c0638bf8a795667/src/components/bulk-email-tool/bulk-email-form/BulkEmailForm.jsx#L307

This line is actually wrong.  The correct check would be ```getConfig().SCHEDULE_EMAIL_SECTION === "true"```.  What it's actually doing now is just checking to see if the variable is empty of not.  Setting it to "False" in a .env file would not have the expected results.

All of this said, we'll just have to live with some inconsistencies in tutor-mfe until everything conforms to the same standard upstream.